src/db.rs

```diff
@@ -720,14 +731,17 @@ mod tests {
         )
         .await
         .unwrap();
+        next_wal_id += 1;
```

Comment:
Nit: mind adding a comment here denoting that this += is for the empty WAL on startup?